### PR TITLE
Refactor dracula theme

### PR DESCRIPTION
**Pull request by Chirikumbrah**
I tried to make dracula scheme more comfortable.

image
normal color is now purple

image
insert cursor color is green now

image
and select is cyan now

image
image
selection colors are also changed

image
image
cursorline and statusline colors are lighter now

image
i changed some colors of variables, types and operators, so now they are nicer to the eyes, imho :)

image
diagnostic info color is cyan now

image
diagnostic warning is now yellow

there are also some changes that u can see in the commit info :)

### CODE REVIEW

**Comment by galearez** [comment]
I was thinking about doing this too. So the only thing I have to say is that the Dracula theme has a big community so I recommend you to read the [Dracula theme specification](https://spec.draculatheme.com/) and maybe you can [contribute there](https://draculatheme.com/contribute) too.

**Comment by Chirikumbrah** [comment]
Nice, thank you!)
I'll do some changes according to the specifications)

**Comment by loewenheim** [comment]
I'm fine with the changes :)

**Review by pickfire — APPROVED** [review]
Thanks for sending the PR.

File: runtime/themes/dracula.toml [hunk 1]
```diff
@@ -1,80 +1,141 @@
 # Author : Sebastian Zivota <loewenheim@mailbox.org>
```
Comment (pickfire) [inline runtime/themes/dracula.toml:1]:
```suggestion
# Author : Sebastian Zivota <loewenheim@mailbox.org>
# Author : Chirikumbrah
```

**Review by pickfire — APPROVED** [review]

**Comment by pickfire** [comment]
Can you please resolve the conflicts?

**Comment by Chirikumbrah** [comment]
Resolved!
Thanks)

**Comment by Chirikumbrah** [comment]
> Can you please resolve the conflicts?

I edited this from the browser on my smartphone and I didn't see the button to commit)
It should be ok now)

**Comment by mo8it** [comment]
@Chirikumbrah This is too much purple for me :(

Thank you for your contribution, especially the selection color is better now, but I think that the main color should stay white.

Sticking to the specification keeps things consistent :)

**Comment by mo8it** [comment]
image

Too much purple which is not a neutral color to me :(

**Comment by Chirikumbrah** [comment]
@mo8it After using NvChad with its chadracula for a while I'm very used to the purple variables. So it seems natural for me :)

But I'm agree with your words about consistency and I'll change the color to white!

Thank you)

**Comment by mo8it** [comment]
You can create a new theme as modified dracula with more purple. That would make us both happy 😄 

Again thank you for the changes, it is just the purple :)

**Comment by Chirikumbrah** [comment]
> You can create a new theme as modified dracula with more purple. That would make us both happy 😄 
That's great idea. I'm thinkin' about it)

**Pull request merged**